cms/bundles/tests/test_views.py

```diff
@@ -300,6 +300,14 @@ def test_preview_release_calendar_page_when_bundle_does_not_exit(self):
         response = self.client.get(reverse("bundles:preview_release_calendar", args=[99999]))
         self.assertEqual(response.status_code, HTTPStatus.NOT_FOUND)
 
+    @override_settings(  # Address race condition in tests caused when calling delete() on a page
+        WAGTAILSEARCH_BACKENDS={
+            "default": {
+                "BACKEND": "wagtail.search.backends.base.SearchBackend",
```

Comment:
Comment: This uses the base barebones `SearchBackend` as explained [here](https://github.com/wagtail/wagtail/wiki/Search-backend-implementation-notes)